src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -1766,6 +1766,19 @@ private static CompletionItem CreateSymbolCompletion(Symbol symbol, Range replac
                     .Build();
             }
 
+            if(symbol is ParameterSymbol parameterSymbol)
```

Comment:
I'm wondering if we actually need to make this specific to just parameters - take a look at the implementation of `BicepHoverHandler.TryGetDescriptionMarkdown`.

Rather than specifically checking for a param symbol, what about just always setting `WithDocumentation` if this function returns a non-null string?